src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/handlers/presubmit/AppealSkeletonBundleGenerator.java

```diff
@@ -34,6 +34,10 @@
     private final DocumentBundler documentBundler;
     private final DocumentHandler documentHandler;
 
+    @Value("${featureFlag.isSaveAndContinueEnabled}")
```

Comment:
Usually we put `@Value` fields to the constructor

Reply:
done